Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -394,6 +401,34 @@ export class Kudu {
         }
     }
 
+    public async zipDeploy(webPackage: string, queryParameters: Array<string>): Promise<any> {
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'POST';
+        httpRequest.uri = this._client.getRequestUri(`/api/zipdeploy`, queryParameters);
+        httpRequest.body = fs.createReadStream(webPackage);
+
+        try {
+            var response = await this._client.beginRequest(httpRequest);
+            tl.debug(`ZIP Deploy response: ${JSON.stringify(response)}`);
+            if(response.statusCode == 200) {
+                tl.debug('Deployment passed');
+                return null;
```

Comment:
Should we return the response body here ?

Reply:
response body is empty string here.